webapp/src/main/scala/io/iohk/atala/mediator/AppUtils.scala

```diff
@@ -83,11 +84,12 @@ object AppUtils {
                 // src := "https://atalaprism.io/images/atala-prism-logo-suite.svg",
```

Comment:
may be remove this or fix it identus

Reply:
I wanted to fix but there is no Identus logo ATM